packages/xpath/vite.config.ts

```diff
@@ -83,6 +96,7 @@ export default defineConfig(({ mode }) => {
 		},
 		define: {
 			TZ: JSON.stringify(timeZoneId),
```

Comment:
This was intended/expected to set the global timezone for tests, does that sound right? Is the issue then that this doesn't work or at least not consistently? Is some of what's at https://github.com/vitest-dev/vitest/issues/1575#issuecomment-1439286286 related? 

If there was an attempted alternative approach to set a consistent timezone that didn't work, I think it'd be helpful to remove it so there isn't ambiguity about the source of truth.

Reply:
Good question

The `define` option in the Vite config is for defining global constants that are replaced at build time or made available as globals during development. The test runner doesn't automatically set the timezone by defining the TZ constant.

We still need the constant to setup the xpath's [evaluator option](https://github.com/getodk/web-forms/blob/4ae10116a9636fcbac044ddd18d4e614dad1df05/packages/xpath/test/helpers.ts#L95) and to get the [default locale](https://github.com/getodk/web-forms/blob/4ae10116a9636fcbac044ddd18d4e614dad1df05/packages/xpath/test/helpers.ts#L362) for mocking the timezone and locale in the [global `beforeEach`](https://github.com/getodk/web-forms/blob/4ae10116a9636fcbac044ddd18d4e614dad1df05/packages/xpath/test/setup.ts#L7)

Reply:
ref: https://vite.dev/config/shared-options.html#define

Reply:
Thanks for the explanation! I'm not super familiar with these test options. So now I'm seeing that the evaluator timezone was being set but there was nothing previously setting the vitest timezone. From what I've read, using a global `beforeEach` seems like a good approach to do that.

Reply:
> Thanks for the explanation! I'm not super familiar with these test options. 

No worries, I'm also learning as I fix this bug